packages/mobile/views/dashboard/drawers/account-switcher/AccountSwitcherDrawer.svelte

```diff
@@ -0,0 +1,65 @@
+<script lang="typescript">
```

Comment:
Because this file is a router, based on the new convention, we should add the word "Router" to the name of the file